checks/caesar.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import File, Test, Error, check
+
+class Vigenere(Test):
```

Comment:
Also, @hljames, this should probably be `class Caesar` instead of `class Vigenere`?